manifests/add-cluster/helm/values.yaml

```diff
@@ -0,0 +1,18 @@
+# serviceAccountName is the name of the ServiceAccount Argo-CD will use to access the cluster
+serviceAccountName: argocd-manager
+
+# ingressUrl is the url used to access the CSDP runtime
+ingressUrl: <ingressUrl>
+
+# csdoToken is used to authenticate against the app-proxy
```

Comment:
```suggestion
# csdpToken is used to authenticate against the app-proxy
```

Reply:
is there a way to use `cf config ... ` cli commands to display the current user's api-key? or should i direct users to just create a new api-key for this purpose? or view their `~/.cfconfig` file?

Reply:
I don't think it's a normal thing a user would like a cli thing to do, to show his secrets. But we should have a command to generate a populated helm values file with all of the values.